backend/src/clothes/clothes.controller.ts

```diff
@@ -0,0 +1,73 @@
+import { Controller, Get, Post, Put, Delete, Body, Param } from '@nestjs/common';
+
+@Controller('test/clothes')
+export class ClothesController {
+  private clothes: Map<string, Object>;
+
+  constructor() {
+    this.clothes = new Map();
+    this.clothes.set((+new Date).toString(16), {
```

Comment:
weak id. It is made from milliseconds. One millisecond is too large for uniqueness